### PR TITLE
grub-mkrelpath: add page

### PR DESCRIPTION
**Pull request by pg99696**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/grub-mkrelpath.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Set the file system path relative to the root which can be used by other GRUB utilities:
+
+`grub-mkrelpath {{path/to/relativepath}}`
+
+- Print the GRUB version number:
+
+`grub-mkrelpath --version`
+
+- Print help page:
```
Comment (kbdharun) [inline pages/linux/grub-mkrelpath.md:14]:
```suggestion
- Display help:
```

File: pages/linux/grub-mkrelpath.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a file system path relative to the root of its file system.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the file system path relative to the root which can be used by other GRUB utilities:
+
+`grub-mkrelpath {{path/to/relativepath}}`
+
+- Print the GRUB version number:
```
Comment (kbdharun) [inline pages/linux/grub-mkrelpath.md:10]:
```suggestion
- Display GRUB version number:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/grub-mkrelpath.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a file system path relative to the root of its file system.
```
Comment (navarroaxel) [inline pages/linux/grub-mkrelpath.md:3]:
```suggestion
> Make a filesystem path relative to the root of its filesystem.
```

File: pages/linux/grub-mkrelpath.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the file system path relative to the root which can be used by other GRUB utilities:
+
+`grub-mkrelpath {{path/to/relativepath}}`
```
Comment (navarroaxel) [inline pages/linux/grub-mkrelpath.md:8]: this should be a directory, right?
```suggestion
`grub-mkrelpath {{path/to/directory}}`
```

File: pages/linux/grub-mkrelpath.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a file system path relative to the root of its file system.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the file system path relative to the root which can be used by other GRUB utilities:
```
Comment (navarroaxel) [inline pages/linux/grub-mkrelpath.md:6]:
```suggestion
- Set the filesystem path relative to the root which can be used by other GRUB utilities:
```

File: pages/linux/grub-mkrelpath.md (hunk 4 above)
Comment (kbdharun) [reply pages/linux/grub-mkrelpath.md:8]: I think we should leave the relative path as it is. 

Ref:

```text
The program grub-mkrelpath makes a file system path relative to the root of its containing file system. For instance, if /usr is a mount point, then:

$ grub-mkrelpath /usr/share/grub/unicode.pf2
‘/share/grub/unicode.pf2’
```

Comment (navarroaxel) [reply pages/linux/grub-mkrelpath.md]: It's ambigous, it's a `path/to/file_or_directory` then?

Comment (pg99696) [reply pages/linux/grub-mkrelpath.md]: > It's ambigous, it's a `path/to/file_or_directory` then?

It makes sense and clear for others. Shall we go with this change @kbdharun and @navarroaxel ?

Comment (kbdharun) [reply pages/linux/grub-mkrelpath.md]: > It's ambigous, it's a `path/to/file_or_directory` then?

I am not sure should we ask @CleanMachine1 ?

Comment (pg99696) [reply pages/linux/grub-mkrelpath.md]: > It's ambigous, it's a `path/to/file_or_directory` then?

@navarroaxel, I have found the following link where the path is mentioned as file name. https://manpages.ubuntu.com/manpages/jammy/man1/grub-mkrelpath.1.html

Should we make it as path_to_file?

File: pages/linux/grub-mkrelpath.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a filesystem path relative to the root of its filesystem.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the filesystem path relative to the root which can be used by other GRUB utilities:
```
Comment (EmilySeville7cfg) [inline pages/linux/grub-mkrelpath.md:6]:
```suggestion
- Set a specific filesystem path relative to the root which can be used by other GRUB utilities:
```

Optional fix: `a specific` may sound better.

File: pages/linux/grub-mkrelpath.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+`grub-mkrelpath {{path/to/directory}}`
+
+- Display GRUB version number:
+
+`grub-mkrelpath --version`
+
+- Display help:
+
+`grub-mkrelpath --help`
```
Comment (EmilySeville7cfg) [inline pages/linux/grub-mkrelpath.md:16]:
```suggestion
- Display help:

`grub-mkrelpath --help`

- Display GRUB version number:

`grub-mkrelpath --version`
```

Optional fix: examples can be reordered.
Hint: to find all pages containing `--help` and `--version` you can use `for f in *.md; do if grep -E -- --version "$f" &> /dev/null && grep -E -- help "$f" &> /dev/null; then echo $f; fi; done` one liner.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- This page should be reviewed by `grub-mkrelpath` users.
- Thanks for a contribution! ❤️

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- This page should be reviewed by `grub-mkrelpath` users.
- Thanks for a contribution! ❤️

**Pull request closed without merging**